core/src/main/java/com/linecorp/armeria/common/Cookie.java

```diff
@@ -99,6 +99,90 @@ static Cookies fromCookieHeader(boolean strict, String cookieHeader) {
         return ServerCookieDecoder.decode(strict, cookieHeader);
     }
 
+    /**
+     * Decodes the specified {@code "Cookie"} header values into a set of {@link Cookie}s.
+     *
+     * @param cookieHeaders the {@code "Cookie"} header values.
+     * @return the decoded {@link Cookie}s.
+     */
+    static Cookies fromCookieHeaders(String... cookieHeaders) {
+        return fromCookieHeaders(true, cookieHeaders);
+    }
+
+    /**
+     * Decodes the specified {@code "Cookie"} header values into a set of {@link Cookie}s.
+     *
+     * @param cookieHeaders the {@code "Cookie"} header values.
+     * @return the decoded {@link Cookie}s.
+     */
+    static Cookies fromCookieHeaders(Iterable<String> cookieHeaders) {
+        return fromCookieHeaders(true, cookieHeaders);
+    }
+
+    /**
+     * Decodes the specified {@code "Cookie"} header values into a set of {@link Cookie}s.
+     *
+     * @param strict whether to validate that the cookie names and values are in the valid scope
+     *               defined in RFC 6265.
+     * @param cookieHeaders the {@code "Cookie"} header values.
+     * @return the decoded {@link Cookie}s.
+     */
+    static Cookies fromCookieHeaders(boolean strict, String... cookieHeaders) {
+        requireNonNull(cookieHeaders, "cookieHeaders");
```

Comment:
We delegate this to the method that takes `Iterable<String>` so we can put the logic into one place.
So could you do this?
```
requireNonNull(cookieHeaders, "cookieHeaders");
return fromCookieHeaders(ImmutableList.copyOf(cookieHeaders));
```